File: Python/Step7/2738.py
import sys

N, M = map(int, sys.stdin.readline().rstrip().split(" "))

ret = [[0 for j in range(M)] for i in range(N)]

for i in range(2):
    for j in range(N):
        ipt = list(map(int, sys.stdin.readline().rstrip().split(" ")))
        
        for k in range(M):
            ret[j][k] += ipt[k]
            
for i in range(N):
    print(" ".join(map(str, ret[i])))

File: Nodejs/backjoonNode/2_dim_array/2738.js
/*
백준 2738번: 2차원 배열
https://www.acmicpc.net/problem/2738
*/

const fs = require('fs');
const table = fs.readFileSync('/dev/stdin').toString().trim().split("\n")
    .map(e => e.split(" ").map(el => parseInt(el)));

const N = table[0][0];
let ret = table.slice(1, 1 + N).map((e, i) => {
    return e.map((el, idx) => {
        return el + table[1+i+N][idx];
    }).join(" ");
}).join("\n");

console.log(ret);